### PR TITLE
Run a chairman process per node

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

**Review by erikd — APPROVED** [review]

**Comment by newhoggy** [comment]
bors merge

**Comment by newhoggy** [comment]
bors merge

File: cardano-node-chairman/test/Test/Cardano/Node/Chairman/ByronShelley.hs [hunk 1]
```diff
@@ -652,6 +653,13 @@ prop_chairman = H.propertyOnce . H.workspace "chairman" $ \tempAbsPath -> do
         )
       )
 
+    return hProcess
+
+  -- Check for chairman success
+  forM_ (L.zip allNodes hChairmanProcesses) $ \(node, hProcess) -> do
```
Comment (coot) [inline cardano-node-chairman/test/Test/Cardano/Node/Chairman/ByronShelley.hs:659]: I think this is not right.  There should be a single `chairman` process connected to `allNodes`, not a `chairman` per node.

Comment (coot) [reply cardano-node-chairman/test/Test/Cardano/Node/Chairman/ByronShelley.hs:659]: Or is this going to be fixed in a subsequent PR?

**Pull request merged**